door-runtime/src/jsMain/kotlin/com/ustadmobile/door/DoorUri.kt

```diff
@@ -9,7 +9,7 @@ import org.w3c.dom.url.URL
 actual class DoorUri(val uri: URL)  {
 
     actual suspend fun getFileName(context: Any): String {
-        TODO("Not yet implemented")
+        TODO("getFileName: Not yet implemented")
```

Comment:
Doesn't the URL have something that could provide this? Otherwise we could take :

if(!path.endsWith("/") {
   path.substringAfterLast("/")
}else {
  path.substring(0, path.size - 1).substringAfterLast("/")
}